<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>IR-RadiationCAM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Created by Charles GAUTHEREAU et Gr√©goire BLANC</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 0px solid 
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="quitButtonConnectForm">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	width : 300px;
	height : 50px;
	border : 2px solid #222;
	border-radius: 10px;
	background-color: white;
	
}
QPushButton:hover{
	background-color: rgb(97, 97, 97);
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connexionButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	width : 300px;
	height : 50px;
	border : 2px solid #222;
	border-radius: 10px;
	background-color: white;
	
}
QPushButton:hover{
	background-color: rgb(97, 97, 97);
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Connexion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 0px solid
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom du groupe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNomGroupe">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	width : 300px;
	height : 50px;
	border : 1px solid #222;
	border-radius: 5px;
	background-color: white;
	
}
QLineEdit:hover{
	background-color: rgb(200, 200, 200);
	color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Mail 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditMail1">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	width : 300px;
	height : 50px;
	border : 1px solid #222;
	border-radius: 5px;
	background-color: white;
	
}
QLineEdit:hover{
	background-color: rgb(200, 200, 200);
	color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Mail 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditMail2">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	width : 300px;
	height : 50px;
	border : 1px solid #222;
	border-radius: 5px;
	background-color: white;
	
}
QLineEdit:hover{
	background-color: rgb(200, 200, 200);
	color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Mail 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditMail3">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	width : 300px;
	height : 50px;
	border : 1px solid #222;
	border-radius: 5px;
	background-color: white;
	
}
QLineEdit:hover{
	background-color: rgb(200, 200, 200);
	color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="errorLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icam/logo icam (1).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/icam.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="validGroupButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	width : 300px;
	height : 50px;
	border : 2px solid #222;
	border-radius: 10px;
	background-color: white;
	
}
QPushButton:hover{
	background-color: rgb(97, 97, 97);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGroup">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border : 2px solid #222;
    border-radius: 8px;
    padding: 1px 18px 5px 20px;
    min-width: 6em;
}</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>Groupe</string>
   </property>
   <item>
    <property name="text">
     <string notr="true">Groupe</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="arrow.qrc"/>
 </resources>
 <connections/>
</ui>
